src/encoding/ascii.ts

```diff
@@ -0,0 +1,21 @@
+import { Context, Recogniser } from '.';
+import match, { Match } from '../match';
+
+export default class Ascii implements Recogniser {
+  name() {
+    return 'ASCII';
+  }
+
+  match(det: Context): Match | null {
+    const input = det.rawInput;
+
+    for (let i = 0; i < det.rawLen; i++) {
+      const b = input[i];
+      if (b > 127) {
```

Comment:
What is your view on 0-31 control characters?

Reply:
They should be excluded actually, I've excluded 0-31 and 127 which match the behavior of `file`.